src/SpecFlow.Dnx/project.json

```diff
@@ -15,7 +14,7 @@
 	},
 
 	"frameworks": {
-		"net40": {
+		"net451": {
```

Comment:
Should probably do something about these monikers. https://blogs.msdn.microsoft.com/dotnet/2015/12/09/support-ending-for-the-net-framework-4-4-5-and-4-5-1/